tests/server/blueprints/institutes/test_institute_views.py

```diff
@@ -733,33 +733,6 @@ def test_gene_variants_filter(app, institute_obj, case_obj):
         assert "POT1" in str(resp.data)
 
 
-def test_gene_variants_no_valid_gene(app, institute_obj, case_obj, mocker, mock_redirect):
```

Comment:
I would suggest to keep this functionality; doesn't have to be very complicated.

Reply:
I guess you could still keep the test now, possibly changing it to check for the error message instead? But, that is optional.

Reply:
The function that is called here is already covered by tests, check [here](https://app.codecov.io/gh/Clinical-Genomics/scout/blob/main/scout/server/blueprints/variants/controllers.py), line 1048. I think it's fine to just remove this function here!